app/src/main/res/values/strings.xml

```diff
@@ -3,23 +3,23 @@
     <string name="action_failed">Action failed</string>
     <string name="add_repository">Add repository</string>
     <string name="address">Address</string>
-    <string name="all_applications">All applications</string>
-    <string name="all_applications_up_to_date">All your applications are up to date</string>
+    <string name="all_applications">All apps</string>
+    <string name="all_applications_up_to_date">All your apps are up-to-date</string>
     <string name="already_exists">Already exists</string>
     <string name="always">Always</string>
     <string name="amoled">Black</string>
     <string name="anti_features">Anti-features</string>
-    <string name="application">Application</string>
-    <string name="application_not_found">Could not find that application</string>
-    <string name="author_email">Author e-mail</string>
-    <string name="author_website">Author website</string>
```

Comment:
These should stay same, as we have project website and author website both, might be confusing.
We can make `Project website` -> `Website`, but not this